test/buildah-bud/buildah-tests.diff

```diff
@@ -1,130 +1,13 @@
-From 47cc8f54d272039bdab389b29c8b2b5bb085e923 Mon Sep 17 00:00:00 2001
+From b948e99cb6cb4765987711e8d8948841f6d3f7e2 Mon Sep 17 00:00:00 2001
 From: Ed Santiago <santiago@redhat.com>
 Date: Tue, 9 Feb 2021 17:28:05 -0700
 Subject: [PATCH] tweaks for running buildah tests under podman
 
 Signed-off-by: Ed Santiago <santiago@redhat.com>
 ---
- tests/bud.bats     | 22 +++++++++++++---------
  tests/helpers.bash | 28 ++++++++++++++++++++++++----
- 2 files changed, 37 insertions(+), 13 deletions(-)
+ 1 file changed, 24 insertions(+), 4 deletions(-)
```

Comment:
You don't need to review this diff file, either. The important part is: there are no longer any diffs to `bud.bats`, only `helpers.bash`.